locales/nl-NL.json

```diff
@@ -239,24 +239,24 @@
         "subscription_error": {
           "clear_error": "Wis error",
           "permission_denied": "Geen toestemming: zet notificaties aan in je browser.",
-          "request_error": "Er is een error tijdens het ophalen van de subscriptie, probeer opnieuw en als de error blijft, raporteer het probleem naar de Elk repository.",
-          "title": "Kon niet subscriben aan push notificaties"
+          "request_error": "Er is een error tijdens het ophalen van het lidmaatschap, probeer opnieuw en als de error blijft, raporteer het probleem naar de Elk repository.",
+          "title": "Kon niet lid worden van de pushnotificaties"
```

Comment:
isn't 'abonnement' or 'abonneren' better?

Reply:
Agree. Abonnement for 'subscriptie' and 'abonneren' for 'lid worden'